sqlstore/src/main/resources/db/migration/V8900__hibernatize_everything.sql

```diff
@@ -1 +1,7 @@
-UPDATE Platform SET name = UPPER(name);
\ No newline at end of file
+UPDATE Platform SET name = UPPER(name);
+
+INSERT INTO StudyType(name) SELECT DISTINCT studyType AS name FROM Study WHERE studyType NOT IN (SELECT name FROM StudyType);
```

Comment:
Since StudyTypes are SRA-defined, it would be better to ensure that the full set is already in the DB, adding missing ones, and not assume that the existing values are good. Migration should fail if there is bad data that does not conform